dockerfiles/platform-build-base/Dockerfile

```diff
@@ -19,6 +19,9 @@
 
 FROM ubuntu:20.04 as build
 ENV DEBIAN_FRONTEND=noninteractive
+ARG BASE_VER
+RUN [ ${BASE_VER} == 7.12.1 ]  # This is version 7.12.1 of the platform-build-base, and anyone building/using it should have set that in BASE_VER
```

Comment:
Is this evaluate to a valid RUN statement?

Reply:
I believe so - do you believe otherwise?